lib/discourse_webauthn.rb

```diff
@@ -68,7 +72,18 @@ def self.challenge(user, secure_session)
   end
 
   def self.rp_id
-    Discourse.current_hostname
+    Rails.env.production? ? Discourse.current_hostname : "localhost"
```

Comment:
Actually I'm quite confuse why this condition is necessary. Wouldn't `Discourse.current_hostname` be `localhost` in the `test` environment?

Reply:
In non-system specs, `Discourse.current_hostname` returns "test.localhost", per https://github.com/discourse/discourse/blob/passkeys-part-1/config/database.yml#L41

System specs have `localhost` as the hostname. 

In current core tests, we stub both `current_hostname` and `base_url`. 

https://github.com/discourse/discourse/blob/main/spec/support/webauthn_integration_helpers.rb#L60-L63